issue_11/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <div id="js-loading"></div><!-- /#js-loading -->
+    <div id="js-ul"></div><!-- /#js-ul -->
```

Comment:
Q
Is there a reason you are using a div?
I thought I should use `<ul>` since the id name is 'js-ul' & the js appends the` <li> `element to the child.